src/routes/events/containers/EventsContainer.js

```diff
@@ -39,6 +41,11 @@ const mapDispatchToProps = (dispatch: Dispatch<StoreActionType>, ownProps) => ({
 
 export default compose(
   translate('events'),
+  connect((state: StateType, ownProps) => ({
```

Comment:
Same as above: Concerned about types :)

Reply:
What exactly do you want me to do here? :thinking:

Reply:
As you resolved all flowfixme this is fine.